.github/workflows/docs.yml

```diff
@@ -5,7 +5,7 @@ name: Deploy mdBook site to Pages
 on:
   # Runs on release
   release:
-    types: [ published ]
+    types: [ released ]
```

Comment:
A possible problem (change?) with this approach: Docs will only start building after we manually publish the release. Previously the docs workflow triggered immediately after the release one successfully created a pre-release.

Reply:
should be fine; do you happen to know if that works for draft releases (or how do they call it) as well?

Reply:
[The docs](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#release) say that 
```
Note: Workflows are not triggered for the created, edited, or deleted activity types for draft releases. When you create your release through the GitHub browser UI, your release may automatically be saved as a draft.
```
So I'm assuming drafts won't trigger anything.